privateca/certificate_authority_subordinate/main.tf

```diff
@@ -51,12 +50,14 @@ resource "google_privateca_certificate_authority" "root_ca" {
   key_spec {
     algorithm = "RSA_PKCS1_4096_SHA256"
   }
+  // Root CA should be valid indefinetly e.g. 99 years
+  lifetime = "${99 * 365 * 24 * 3600}s"
 }
 
 resource "google_privateca_certificate_authority" "default" {
```

Comment:
let's also replace `"default"` here with `"sub_ca"`, similar to what you did for the root

Reply:
done